packages/components/pie-assistive-text/src/index.ts

```diff
@@ -11,10 +18,39 @@ const componentSelector = 'pie-assistive-text';
 
 /**
  * @tagname pie-assistive-text
+ * @slot - Default slot
  */
 export class PieAssistiveText extends LitElement implements AssistiveTextProps {
+    @property()
+    @validPropertyValues(componentSelector, variants, 'default')
+    public variant?: AssistiveTextProps['variant'] = 'default';
+
+    /**
+ * Renders the assistive-text icon content.
+ * @private
+ */
+    private renderContent (): TemplateResult {
+        const { variant } = this;
+        return html`
+            ${variant === 'success' ? html`<icon-check-circle class="c-assistiveText-icon" size="s" />` : nothing}
+            ${variant === 'error' ? html`<icon-alert-circle class="c-assistiveText-icon" size="s" />` : nothing}`;
+    }
+
     render () {
-        return html`<h1 data-test-id="pie-assistive-text">Hello world!</h1>`;
+        const {
+            variant,
+        } = this;
+
+        return html`
+        <p 
+            class="c-assistiveText"
+            data-test-id="pie-assistive-text"
+            variant=${ifDefined(variant)}>
+            <span class="c-assistiveText-content">
```

Comment:
i might be missing something but do you think we can update the markup to be like

```
<p
            class="c-assistiveText"
            data-test-id="pie-assistive-text"
            variant=${ifDefined(variant)}>
             ${this.renderContent()}
            <slot></slot>
             </p>

```

then move the styles of "c-assistiveText-content" to the parent. this way we can use only one element for the component and encapsulate things in one place. 

also in case it helps, you can also set the size of the icon and its display value using the following css vars. 

```
    --icon-display-override: block;
    --icon-size-override: 20px;
    --icon-size-override: 20px;
```

Reply:
thanks @raoufswe I'll look into this! ❤️ I personally prefer using the `size="s"` prop in `pie-icons-webc` than declaring the icon size variables, but I'll try with the override and removing the span 🤞

Reply:
works without the span 🎉 thank you!